templates/zerver/help/send-group-pm.md

```diff
@@ -0,0 +1,72 @@
+# Group Private Messages
+Users can send group private messages in Zulip.
+## There are three ways to send a Group Private Message:
+### One:
```

Comment:
Maybe `send private group messages` instead of `send group private messages`? Maybe merge lines 2 and 3: Users can send private group messages in Zulip through one of three ways:
`### First`